docs/source/tutorial/detection.rst

```diff
@@ -0,0 +1,176 @@
+The more description can be found here (:meth:`chainercv.links.SSD.predict`).
+Inference on these models runs smoothly by downloading necessary pre-trained weights from the internet automatically.
+
+.. code-block:: python
+
+    from chainercv.visualizations import vis_bbox
+    from chainercv.datasets import VOCBboxDataset
+    from chainercv.datasets import voc_bbox_label_names
+    from chainercv.links import SSD300
+    import matplotlib.pyplot as plt
+    dataset = VOCBboxDataset(year='2007', split='test')
+    img, _, _ = dataset[0]
+    model = SSD300(pretrained_model='voc0712')
+    bboxes, labels, scores = model.predict([img])
+    vis_bbox(img, bboxes[0], labels[0], scores[0],
+             label_names=voc_bbox_label_names)
+    plt.show()
+
+.. figure:: ../../image/detection_tutorial_link.png 
+    :scale: 60%
+    :align: center
+
+The above example puts together functinality of detection link.
+It instantiates SSD300 model with weights trained on VOC 2007 and VOC 2012 datasets.
+The model runs prediction using :meth:`predict`, and the ouptus are visualized using
+:func:`chainercv.visualizations.vis_bbox`.
+Note that in this case, confidence scores are visualized together with other data.
+
+
+Detection Evaluator
+-------------------
+ChainerCV provides functionalities that make evaluating detection links easy.
+They are provided at two levels: evaluator extensions and evaluation functions.
+
+Evaluator extensions such as :class:`chainercv.extensions.DetectionVOCEvaluator` inherit from :class:`chainer.training.extensions.Evaluator`, and have similar interface.
+They are initialized by taking an iterator and a network that carries out prediction with method :meth:`predict`.
+When this class is called (i.e. :meth:`DetectionVOCEvaluator.__call__`), several actions are taken.
+First, it iterates over a dataset based on an iterator.
+Second, the network makes prediction using the images collected from the dataset.
+Last, an evaluation function is called with the ground truth annotations and the prediction results.
+
+In contrast to evaluators that hide details,
+evaluation functions such as :meth:`chainercv.evaluations.eval_detection_voc`
+are provided for those who need finer level of control.
+These functions take the ground truth annotations and prediction results as arguments
+and return measured performance.
+
+Here is a simple example that uses a detection evaluator.
+
+.. code-block:: python
+
+   from chainer.iterators import SerialIterator
+   from chainer.datasets import SubDataset
+   from chainercv.datasets import VOCBboxDataset
+   from chainercv.links import SSD300
+   from chainercv.extensions import DetectionVOCEvaluator
+   from chainercv.datasets import voc_bbox_label_names
+
+   # Only use subset of dataset so that evaluation finishes quickly.
+   dataset = VOCBboxDataset(year='2007', split='test')
+   dataset = dataset[:5]
+   it = SerialIterator(dataset, 1, repeat=False, shuffle=False)
```

Comment:
`batchsize` should be more than one. Some user will think `DetectionVOCEvaluator` does not support batched images.